diff --git a/blocks/ecc-dashboard/ecc-dashboard.js b/blocks/ecc-dashboard/ecc-dashboard.js
index 17f6090..0e1e336 100644
--- a/blocks/ecc-dashboard/ecc-dashboard.js
+++ b/blocks/ecc-dashboard/ecc-dashboard.js
@@ -535,6 +535,7 @@ export default async function init(el) {
   const devMode = urlParams.get('devMode');
 
   const config = readBlockConfig(el);
+  config['create-form-url'] = config['create-form-url'] ?? '/event/create/t3';
   el.innerHTML = '';
   const profile = window.bm8r.get('imsProfile');
 
